src/Microsoft.DotNet.Darc/Darc/Operations/Operation.cs

```diff
@@ -76,7 +77,7 @@ protected Operation(ICommandLineOptions options, IServiceCollection? services =
         services.TryAddSingleton<IAzureDevOpsClient>(s =>
             s.GetRequiredService<AzureDevOpsClient>()
         );
-
+        services.TryAddSingleton<IRemoteTokenProvider>(new RemoteTokenProvider(options.AzureDevOpsPat, options.GitHubPat));
```

Comment:
ah, this isn't good, I need to take local settings into consideration too

Reply:
nevermind, it's good, we overwrite the options with localSettings if they're available

Reply:
I think you're right - we need to take those into account and we do that below in `options.InitializeFromSettings(Logger);`